internal/policy/verify.go

```diff
@@ -599,22 +716,26 @@ func getCommits(repo *gitinterface.Repository, entry *rsl.ReferenceEntry) ([]git
 	return repo.GetCommitsBetweenRange(entry.TargetID, priorRefEntry.TargetID)
 }
 
-func verifyGitObjectAndAttestations(ctx context.Context, policy *State, target string, gitID gitinterface.Hash, authorizationAttestation *sslibdsse.Envelope, approverKeyIDs *set.Set[string]) (string, error) {
+func verifyGitObjectAndAttestations(ctx context.Context, policy *State, target string, gitID gitinterface.Hash, authorizationAttestation *sslibdsse.Envelope, approverKeyIDs *set.Set[string], verifyMergeable bool) (string, bool, error) {
 	verifiers, err := policy.FindVerifiersForPath(target)
 	if err != nil {
-		return "", err
+		return "", false, err
 	}
 
-	return verifyGitObjectAndAttestationsUsingVerifiers(ctx, verifiers, gitID, authorizationAttestation, approverKeyIDs)
+	return verifyGitObjectAndAttestationsUsingVerifiers(ctx, verifiers, gitID, authorizationAttestation, approverKeyIDs, verifyMergeable)
 }
 
-func verifyGitObjectAndAttestationsUsingVerifiers(ctx context.Context, verifiers []*Verifier, gitID gitinterface.Hash, authorizationAttestation *sslibdsse.Envelope, approverKeyIDs *set.Set[string]) (string, error) {
+func verifyGitObjectAndAttestationsUsingVerifiers(ctx context.Context, verifiers []*Verifier, gitID gitinterface.Hash, authorizationAttestation *sslibdsse.Envelope, approverKeyIDs *set.Set[string], verifyMergeable bool) (string, bool, error) {
 	if len(verifiers) == 0 {
 		// This target is not protected by gittuf policy
-		return "", nil
+		return "", false, nil
```

Comment:
> @wlynch: Everytime I read this it throws me for a loop because we're failing open. I wonder if we should allow restricting this somewhere to require at least 1 verifier. 🤔

That'd be a bit strange if someone were to `verify-ref <random unprotected feature ref>`. I agree on failing open, though...